mlir/test/Dialect/MIOpen/lowering_padding_kernel.mlir

```diff
@@ -46,7 +48,7 @@ func @miopen_conv2d_kcyx_nchw_nkhw_no_extra_padding(%filter : memref<1x128x64x3x
 // CHECK-NEXT:  {{miopen.transform.*{extraPad = "false", gemmM_extra = 0 : i32, gemmN_extra = 0 : i32,.*}.*memref.*memref}}
 // CHECK-NEXT:  {{miopen.gridwise_gemm.*{.*}.*memref.*memref.*memref}}
 
-func @miopen_conv2d_kcyx_nchw_nkhw_partial_padding_kernel(%filter : memref<32x4x2x3x3xf32>, %input : memref<128x32x2x11x11xf32>, %output : memref<128x32x4x9x9xf32>) {
+func @miopen_conv2d_kcyx_nchw_nkhw_partial_padding_kernel(%filter : memref<32x128x2x3x3xf32>, %input : memref<128x32x2x11x11xf32>, %output : memref<128x32x128x9x9xf32>) {
```

Comment:
the dimension of these memrefs should not be changed

Reply:
gemmM, gemmN padding not support yet ,turn off it until function complete